packages/spectral/src/serverTypes/convertIntegration.ts

```diff
@@ -624,6 +626,10 @@ export const convertConfigVar = (
       stableKey,
       dataType: "connection",
       useScopedConfigVar: stableKey,
+      meta: {
```

Comment:
Is it safe to always set both of these values to false? Is there a need to check the original values here and carry them over?